samples/bluetooth/mesh/light/README.rst

```diff
@@ -144,6 +147,22 @@ To review the required configuration alterations, open and inspect the two files
 For more information about using configuration overlay files, see :ref:`zephyr:important-build-vars` in the Zephyr documentation.
 For more information about selecting a sysbuild configuration file, see the Sysbuild Kconfig file section on the :ref:`zephyr:sysbuild` page in the Zephyr documentation.
 
+nRF53 Development Kit
+*********************
+
+To enable the DFU feature for the nRF53 development kit, set :makevar:`SB_CONF_FILE` to :file:`sysbuild-dfu.conf` and set :makevar:`EXTRA_CONF_FILE` to :file:`overlay-dfu.conf` when building the sample.
```

Comment:
Just a suggestion, but this part is common to all mentioned DKs. It could be sufficient to have a common section with this?